src/raw/client.rs

```diff
@@ -864,6 +898,16 @@ impl<PdC: PdClient> Client<PdC> {
     }
 }
 
+#[derive(Clone)]
+struct ScanInnerArgs {
+    start_key: Key,
+    range: BoundRange,
```

Comment:
```suggestion
    end_key: Option<Key>,
```

As `start_key` and `range.from` are always the same, I think we can keep only one from them. And as it's easier to convert `start_key` and `end_key` to `range` than opposite, I suggest to have `start_key` and `end_key` in `ScanInnerArgs`.

Rest LGTM.

Reply:
Changed! Thanks for the review.